invadedhouse/commands.py

```diff
@@ -93,13 +94,13 @@ def interpret_command(self, cmd: str):
                 print('Nowhere! You\'re stuck now! Mwahahaha.')
             else:
                 print('Possible paths:')
-                for exit in exits:
-                    print(exit)
+                for room in exits:
+                    print(room)
```

Comment:
Python doesn't like us using the keyword `exit` since that is a python command to quit the running program. So I changed this variable name to `room`